src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -1522,19 +1531,27 @@
   "com.minecolonies.coremod.gui.chat.cancel": "Try cancel that request!",
   "com.minecolonies.coremod.gui.chat.fulfill": "More details please!",
   "com.minecolonies.coremod.gui.research.title": "Research Branches",
-  "com.minecolonies.coremod.research.requirement.building.level": "Requires %s at least level %d!",
+  "com.minecolonies.coremod.gui.research.tier.header": "Level %d (%d hr)",
+  "com.minecolonies.coremod.research.requirement.building.level": "Requires %s(s) totalling at least %d levels!",
```

```suggestion
  "com.minecolonies.coremod.research.requirement.building.level": "Requires %s(s) totaling at least %d levels!",
```
This is the json for American English, not British English 😄